app/code/Magento/InventoryReservationCli/Model/GetReservationsList.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryReservationCli\Model;
+
+use Magento\InventoryReservationCli\Model\ResourceModel\GetListReservations;
+
+class GetReservationsList
```

Comment:
why do we need this class if all what it does is proxies method call to `$this->getListReservations->execute();` ?

Reply:
hi @maghamed, usually i never call a resource model class directly. So i created a class model